python/nav/django/settings.py

```diff
@@ -171,10 +171,14 @@
         'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
         'LOCATION': '/tmp/nav_cache',
         'TIMEOUT': '60',
-    }
+    },
+    'sortedstats': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
+        'LOCATION': '/tmp/nav_cache',
+        'TIMEOUT': '900',
+    },
```

Comment:
The comment that precedes this block needs an update, since it's no longer the case that only the report subsystem uses the Django cache.

Reply:
Updated comment